code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -940,7 +940,7 @@
 /obj/effect/clockwork/sigil/vitality/examine(mob/user)
 	..()
 	if(is_servant_of_ratvar(user) || isobserver(user))
-		user << "<span class='[vitality ? "inathneq_small":"alloy"]'>It is storing [vitality] units of vitality.</span>"
+		user << "<span class='[vitality ? "inathneq_small":"alloy"]'>It is storing [ratvar_awakens ? "INFINITY":"[vitality]"] units of vitality.</span>"
```

Comment:
That should read "INFINITE", not "INFINITY". You could also opt for something better, such as "storing an immeasurable amount of power" or such.